.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+src/.DS_Store
```

Comment:
just add `.DS_store` and it will exclude all the .DS_store. because there are still some DS_store remaining like src/components and src/images/.DS_store